app/helpers/reviews_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ReviewsHelper
+end
```

Comment:
What's going on with this helper? Are you actually using it somewhere? Was it created perchance by a generator?